build.gradle

```diff
@@ -1,7 +1,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.support_libraries_version = '26.1.0'
+    ext.support_libraries_version = '27.1.1'
```

Comment:
btw, I discovered you can actually define these values inside the `dependencies { }` block in the other file, e.g.
```groovy
dependencies {
    def support_libraries_version = '27.1.1'
    implementation "...:$support_libraries_version"
```

For a code example, see https://github.com/mcomella/NonContactCallBlocker/blob/dce463a70a09932b5e7ee7a199215a184a80b1fe/app/build.gradle#L36

The downside to this approach is that you can only use the value in the dependencies block, which is fine for most deps but not kotlin_version (which is used by another plugin).

Reply:
> The downside to this approach is that you can only use the value in the dependencies block, which is fine for most deps but not kotlin_version (which is used by another plugin).

Yeah, and we may want to separate our code better by using multiple gradle modules in the future.

Reply:
Btw. in the Android components repository (where we have a bunch of gradle modules) I'm planning to move the whole dependency definition into the root build.gradle.

Something like:
```Groovy
ext.dependencies = [
    appcompat: "com.android.support:appcompat-v7:27.1.1"
    // ...
]
```

And then in the modules that use the dependency something like:
```Groovy
implementation rootProject.ext.dependencies['appcompat']
```

That's something I want to look into once we switch to the `androidx` builds of the support libraries where every library is going to have its own version.